protocol/x/clob/memclob/memclob.go

```diff
@@ -1073,6 +1087,13 @@ func (m *MemClobPriceTimePriority) RemoveAndClearOperationsQueue(
 	ctx sdk.Context,
 	localValidatorOperationsQueue []types.InternalOperation,
 ) {
+	defer telemetry.MeasureSince(
```

Comment:
LMK if anyone prefers I use `ModuleMeasureSince` instead of `MeasureSince`. IMO think it's clearer to not use module name as a label.

Also can update stat names like `RemoveAndClearOperationsQueue` to include the module name as a prefix to avoid unnecessary string concatenation, but these stats aren't in the hot path so I _think_ this is fine.